Source/WebKit/Platform/haiku/WorkQueueHaiku.cpp

```diff
@@ -1,44 +1,147 @@
 /*
-    Copyright (C) 2014 Haiku, Inc
-
-    This library is free software; you can redistribute it and/or
-    modify it under the terms of the GNU Library General Public
-    License as published by the Free Software Foundation; either
-    version 2 of the License, or (at your option) any later version.
-
-    This library is distributed in the hope that it will be useful,
-    but WITHOUT ANY WARRANTY; without even the implied warranty of
-    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
-    Library General Public License for more details.
-
-    You should have received a copy of the GNU Library General Public License
-    along with this library; see the file COPYING.LIB.  If not, write to
-    the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
-    Boston, MA 02110-1301, USA.
+ * Copyright (C) 2016 Haiku, Inc. All rights reserved.
```

Comment:
I'm curious about this file, where does it come from? Why is the copyroght year 2016?

Reply:
am sorry @pulkomandy i was trying to fix workqueue initially it had not implemented functions. But when i saw about workqueue i found it should be under web template framework as it is the one which handles threads and stuff. so got confused little bit 😅

Reply:
i will restore this part right now!

Reply:
Ah, yes, this was moved from WebKit to WTF. So this file can be deleted, then?

Reply:
yeah i guess workqueue is handled by wtf so no need in webkit right?